ee/clickhouse/queries/stickiness/stickiness_actors.py

```diff
@@ -0,0 +1,43 @@
+from typing import Dict, Optional, Tuple
+
+from ee.clickhouse.queries.actor_base_query import ActorBaseQuery
+from ee.clickhouse.queries.stickiness.stickiness_event_query import StickinessEventsQuery
+from posthog.models.entity import Entity
+from posthog.models.filters import Filter
+from posthog.models.filters.mixins.utils import cached_property
+from posthog.models.filters.stickiness_filter import StickinessFilter
+from posthog.models.team import Team
+
+
+class ClickhouseStickinessActors(ActorBaseQuery):
+    entity: Entity
+    _filter: StickinessFilter
+
+    def __init__(self, team: Team, entity: Optional[Entity], filter: StickinessFilter):
+        if not entity:
+            raise ValueError("Entity is required")
```

Comment:
Should this be a validationError or is a 500 warranted?

Reply:
just removed this check. the init shouldn't take an optional